src/__main__.py

```diff
@@ -116,13 +123,33 @@ async def list(ctx, code=None):
     # Command to create, modify permissions for, or delete private study groups.
     @bot.command(pass_context=True)
     async def group(ctx, *args):
-        
+        '''
+        Creates private study groups.
+
+        !group create [group_name] @users - Creates a private study group and invites the mentions.
+        !group delete [group_name]        - Deletes a private study group you are in.
+        !group add    [group_name] @users - Adds mentioned users to a study group you are in.
+        '''
+        # If the user attempts to append the naming convention...
+        if args[1].endswith('-study-group'):
```

Comment:
Good use of the study-group suffix, but maybe allow the command to go through and give the user a warning that they don't need to specify the "-study-group" suffix.